packages/phone-number-privacy/signer/test/integration/legacypnp.test.ts

```
@@ -1085,40 +1085,30 @@ describe('legacyPNP', () => {
           const authorization = getPnpRequestAuthorization(req, PRIVATE_KEY1)
           const res = await sendRequest(req, authorization, SignerEndpoint.LEGACY_PNP_SIGN)
 
-          expect(res.status).toBe(200)
-          expect(res.body).toMatchObject<SignMessageResponseSuccess>({
-            success: true,
+          expect(res.status).toBe(500)
+          expect(res.body).toMatchObject<SignMessageResponseFailure>({
+            success: false,
             version: res.body.version,
-            signature: expectedSignature,
-            performedQueryCount: 1,
+            performedQueryCount: -1,
             totalQuota: expectedQuota,
             blockNumber: testBlockNumber,
-            warnings: [
-              ErrorMessage.DATABASE_GET_FAILURE,
-              ErrorMessage.FAILURE_TO_GET_PERFORMED_QUERY_COUNT,
-            ],
+            error: ErrorMessage.DATABASE_GET_FAILURE,
           })
 
           spy.mockRestore()
-
-          // check DB state: performedQueryCount was still incremented and request was stored
-          expect(
-            await getPerformedQueryCount(db, ACCOUNT_ADDRESS1, rootLogger(config.serviceName))
-          ).toBe(expectedQuota + 1)
-          expect(await getRequestExists(db, req, rootLogger(config.serviceName))).toBe(true)
         })
 
-        it('Should return 200 w/ warning on blockchain totalQuota query failure', async () => {
+        it('Should return 200 w/ warning on blockchain totalQuota query failure when shouldFailOpen is true', async () => {
           // deplete user's quota
```

optional nit: explicitly set shouldFailOpen to true or assert that it's true, just to prevent or fail early if default settings change